internal/dev_server/model/project.go

```diff
@@ -90,15 +91,21 @@ func SyncProject(ctx context.Context, projectKey string) (Project, error) {
 		return Project{}, err
 	}
 
-	project.FlagState = flagsState
+	project.AllFlagsState = flagsState
 	project.LastSyncTime = time.Now()
+
 	updated, err := store.UpdateProject(ctx, *project)
 	if err != nil {
 		return Project{}, err
 	}
 	if !updated {
 		return Project{}, errors.New("Project not updated")
 	}
+
+	GetObserversFromContext(ctx).Notify(SyncEvent{
+		ProjectKey:    project.Key,
+		AllFlagsState: project.AllFlagsState,
+	})
```

Comment:
Issuing the event in `SyncProject` here.

Reply:
```suggestion
	allFlagsWithOverrides, err := project.GetFlagStateWithOverridesForProject(ctx)
	if err != nil {
		return Project{}, errors.Wrapf(err, "unable to get overrides for project, %s", projectKey)
	}

	GetObserversFromContext(ctx).Notify(SyncEvent{
		ProjectKey:    project.Key,
		AllFlagsState: allFlagsWithOverrides,
	})
```

Reply:
Just realized we need to apply the overrides too.